hera/hooks/useClientGameAction.tsx

```diff
@@ -16,6 +31,7 @@ import toClientGame, {
   ClientGame,
 } from '@deities/hermes/game/toClientGame.tsx';
 import { useCallback } from 'react';
+import EvaluationWorker from '../editor/workers/evaluation?worker';
```

Comment:
We should probably move this file elsewhere since it is now used directly in this hook, and not just within the editor.

Reply:
Any suggestion? Will `hera/workers/evaluation.tsx` is good ?

Reply:
It's not really only going to be used for evaluation, it's also used for running any local game actions. Maybe `hera/workers/gameAction.tsx`?

Reply:
Yeahh, right! I'm thinking this just for evaluation but actually this can be from anywhere else.